README.md

```diff
@@ -286,6 +286,32 @@ expect('hello world').toExclude('goodbye')
 expect('hello world').toNotContain('goodbye')
 ```
 
+### toIncludeKeys
+> `expect(object).toIncludeKeys(keys, [hasKey], [message])`<br>
```

Comment:
For consistency with other methods, also someone might want to use a check which includes properties in the prototype chain. The `has` property only checks immediate properties.